statsrunner/loop.py

```diff
@@ -70,20 +70,25 @@ def is_humanitarian_activity(activity, version):
                     humanitarian_sectors_dac_5_digit = ['72010', '72040', '72050', '73010', '74010']
                     humanitarian_sectors_dac_3_digit = ['720', '730', '740']
 
+                    # Set the correct vocabulary code for the version that this activity is defined at
+                    vocab_code_dac_5_digit = "DAC" if version in ["1.01", "1.02", "1.03", "1.04", "1.05"] else 1
+                    vocab_code_dac_3_digit = "DAC-3" if version in ["1.01", "1.02", "1.03", "1.04", "1.05"] else 2
+
                     # ensure we are dealing with an activity
                     if activity.tag != 'iati-activity':
                         return False
 
                     is_humanitarian_by_attrib = 1 if (version in ['2.02']) and ('humanitarian' in activity.attrib) and (activity.attrib['humanitarian'] in ['1', 'true']) else 0
-                    is_humanitarian_by_sector_5_digit = 1 if set(activity.xpath('sector[@vocabulary="1" or not(@vocabulary)]/@code')).intersection(humanitarian_sectors_dac_5_digit) else 0
-                    is_humanitarian_by_sector_3_digit = 1 if set(activity.xpath('sector[@vocabulary="2"]/@code')).intersection(humanitarian_sectors_dac_3_digit) else 0
+                    is_humanitarian_by_sector_5_digit = 1 if set(activity.xpath('sector[@vocabulary="{}" or not(@vocabulary)]/@code'.format(vocab_code_dac_5_digit))).intersection(humanitarian_sectors_dac_5_digit) else 0
```

Comment:
The squiggle brackets `{}` should have a `0` in to better indicate that they accept the first argument to `format()`